st2tests/st2tests/fixtures/packs/configs/dummy_pack_22.yaml

```diff
@@ -0,0 +1,2 @@
+---
+  key_with_no_secret_and_no_default: "Any Value"
```

Comment:
Please get rid of the end of line character.

Reply:
Done